runelite-client/src/main/java/net/runelite/client/plugins/dailytaskindicators/DailyTasksPlugin.java

```diff
@@ -61,7 +63,10 @@
 	@Inject
 	private ChatMessageManager chatMessageManager;
 
-	private boolean hasSentHerbMsg, hasSentStavesMsg, hasSentEssenceMsg;
+	private boolean hasSentHerbMsg, hasSentStavesMsg, hasSentEssenceMsg, hasSentSandMsg, hasSentEctoMsg;
+	private int prevHerbVarbVal, prevStavesVarbVal, prevEssVarbVal, prevSandVarbVal, prevEctoVarbVal;
+
+	private final Set<Integer> MEMBER_WORLDS = Sets.newHashSet(1, 8, 16, 26, 35, 81, 82, 83, 84, 85, 93, 94, 117);
```

Comment:
Is this variable named correctly? This set contains the f2p worlds, but not the members worlds. `s/MEMBER/F2P/`?

Reply:
Good catch, my fault, pushing the changed variable now